src/psalter.php

```diff
@@ -181,6 +188,11 @@ function ($arg) use ($valid_long_options, $valid_short_options) {
     new Psalm\Internal\Provider\ClassLikeStorageCacheProvider($config)
 );
 
+if (array_key_exists('list-supported-issues', $options)) {
+    echo 'Supported issues: ' . implode(',', ProjectAnalyzer::getSupportedIssuesToFix()) . PHP_EOL;
```

Comment:
Probably this should be less verbose:
```suggestion
    echo implode(',', ProjectAnalyzer::getSupportedIssuesToFix()) . PHP_EOL;
```